ross/plotly_theme.py

```diff
@@ -0,0 +1,377 @@
+import plotly.graph_objects as go
+import plotly.io as pio
+
+# tableau colors
+colors_dict = {
+    "blue": "#1f77b4",
+    "orange": "#ff7f0e",
+    "green": "#2ca02c",
+    "red": "#d62728",
+    "purple": "#9467bd",
+    "brown": "#8c564b",
+    "pink": "#e377c2",
+    "gray": "#7f7f7f",
+    "olive": "#bcbd22",
+    "cyan": "#17becf",
+}
```

Comment:
Wouldn't it be simpler to import this palette directly from plotly?
I mean:
```
from plotly.express import colors
tableau_colors = colors.qualitative.T10
```

Reply:
We could do that, but the problem is that `colors.qualitative.T10` returns just a list with hex values and I wanted to have a dictionary so that in other parts of the code we could use colors_dict['blue'] instead of `tableau_colors[0]`.